package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pool-request",
-  "version": "0.0.0",
+  "version": "0.2.1",
```

Comment:
well i added two changes and one tiny patch, i was gonna update major version to 1 but that would denote that pool request is out of alpha and beta and prod ready. What do you think?